api/mapping/views.py

```diff
@@ -1890,3 +1912,93 @@ def scanreport_admin_page(request, pk):
         args["is_admin"] = False
 
     return render(request, "mapping/admin_scanreport_form.html", args)
+
+
+@login_required
+def scanreport_table_page(request, pk):
+    args = {}
+    scan_report = None
+    scan_report_name = None
+    scan_report_table = None
+    data_dictionary = None
+
+    scan_report = ScanReport.objects.get(id=pk)
```

Comment:
There are a couple of outstanding issues here (and they may be present at the values/fields level as well).

1. 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/11610738/162699463-52ae1a5c-0a17-41fc-a336-0b4b959a5f57.png">

Here we're in SR 317, and the table `cdw.csv` links to the URL `http://localhost:8080/fields/?search=1248`. However, if I try to access that table through the new URL:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/11610738/162699733-7f09eb87-0bb3-4de8-9330-7303faa5e8b8.png">
Having a bit of a play around, I seem to be able to access `317/tables/1` to `317/tables/417`, but no higher. We have over 5000 entries in the `mapping_scanreporttable` table. So something fishy is going on.

2. Relatedly, if I put in an SR ID and table ID that aren't even paired together, then I get a rather nonsensical result:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/11610738/162700283-596e68bc-a9a8-4134-9fdc-e5f15eaa3788.png">
This is the table with global ID 1, so is unrelated to SR 317. But the page merrily loads, and the breadcrumbs along the top happily give the impression that this is a table from `EAVE-II-v2` when this is not the case. This should be returning 404 (or other if authentication gets in the way before that point).